result/bgpprefix.yaml

```diff
@@ -249,3 +261,17 @@ components:
         as_path:
           $ref: './resultbgpaspath.yaml#/components/schemas/Result.BgpAsPath'
           x-field-uid: 7
+        local_preference:
+          description: >-
+            The local preference is a well-known attribute and the value is used for route selection.
+            The route with the highest local preference value is preferred.
```

Comment:
Not sure we want to add more comment: copied from rfc "A BGP speaker MUST NOT include this attribute in UPDATE messages it
   sends to external peers, except in the case of BGP Confederations
   [RFC3065]"

Reply:
Since this is related to Received attributes, I wanted to avoid transmission related requirements for this attribute. I felt better place to include this particular transmission characteristic might have been in Local preference attribute setting in Route attributes ( in Route range )